fastlane/Fastfile

```diff
@@ -10,27 +10,26 @@
 #     https://docs.fastlane.tools/plugins/available-plugins
 #
 
-import("#{File.expand_path(File.dirname(__FILE__))}/CommonFastfile")
-
 default_platform(:ios)
 
-ENV["REPO_NAME"] = "purchases-ios"
-ENV["FILES_TO_UPDATE_VERSION_STRING"] = "
-  ../RevenueCat.podspec,
-  ../Sources/Misc/SystemInfo.swift,
-  ../.jazzy.yaml,
-  ../.version,
-  ../scripts/docs/index.html,
-  ../Sources/Misc/SystemInfo.swift
-"
-ENV["FILES_TO_UPDATE_VERSION_WITHOUT_PRERELEASE_MODIFIERS_STRING"] = "
-  ../Sources/Info.plist,
-  ../Tests/BackendIntegrationTestApp/Info.plist,
-  ../Tests/BackendIntegrationTests/Info.plist,
-  ../Tests/UnitTests/Info.plist,
-  ../Tests/UnitTestsHostApp/Info.plist
-"
-ENV["GITHUB_RELEASE_UPLOAD_ASSETS"] = "RevenueCat.framework.zip,RevenueCat.xcframework.zip"
```

Comment:
For those wondering like me, this is defined inline now since it's only used in one place 👍🏻